src/shared/shapes/types.d.ts

```diff
@@ -0,0 +1,4 @@
+type ShapeProps = {
```

Comment:
doesn't it should be exported?

Reply:
No, it does not. I created `d.ts` file. We don't have to export all definitions that were declared there. However, I have rewritten this file to simple `.ts` because I'm not sure that it is a good idea, hmm... We have to investigate the possibility to use a `d.ts`.

Reply:
oh, I see. maybe, creating declaration files is a good idea and we should use it everywhere for other types. But we should check - on which scope these type values are exported (global or only for items in the same folder) and if these declaration files do not cause conflicts with other types which have the same name. (StyleProps etc.)